intelligence/ts/src/engines/remoteEngine/chat.ts

```diff
@@ -270,7 +277,18 @@ async function processChunk(
   }
 
   if (isFinalChunk(parsed)) {
-    return { ok: true, message: { role: 'assistant', content: '' } };
+    return {
+      ok: true,
+      message: {
+        role: 'assistant',
```

Comment:
Do we need this role here?

Reply:
Yes, the type requires either `system`, `assistant`, or `user`